crates/ra_ide/src/lib.rs

```diff
@@ -328,9 +329,9 @@ impl Analysis {
     pub fn inlay_hints(
         &self,
         file_id: FileId,
-        max_inlay_hint_length: Option<usize>,
+        inlay_hint_opts: &InlayHintOptions,
```

Comment:
Note to self: this is actually a better design than what we have for completion, where we use ambient feature flag. Completions should accept `CompletionOptions` as an argument, so that caller can pick options in the most flexible way.

Reply:
This actually would allow us to nicely test arg snippets, as requested by @kjeremy :)